MultiTable/src/RealtimeChangeHandler.js

```diff
@@ -22,8 +24,12 @@ function onSampleRemove(data, sample) {
     if (subjectGroup) {
       subjectGroup.removeSample(sample);
       const subject = subjectGroup.getSubjectForSample(sample);
-      const index = subject.samples.findIndex(s => (s.name === sample.name));
-      subject.samples.splice(index, 1);
+      if (subject) {
+        const index = subject.samples.findIndex(s => (s.name === sample.name));
```

Comment:
here, too -- protect against _possibility_ that subjects exists but does not have samples array?